Readme.md

```diff
@@ -4,27 +4,34 @@
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 
-# Grid for Blazor - Customize the appearance of elements including row and cell values based on custom conditions
+# Grid for Blazor - Customize cell appearance based on custom conditions
 
-The [Blazor Grid](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid) allows you to customize the appearance of individual UI elements based on custom conditions (conditional formatting).
+The example demonstrates how to customize appearance of the [DevExpress Blazor Grid](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid) elements based on custom conditions.
 
-This example customizes the appearance of Blazor Grid cells/rows in the following manner:
- 
-* Highlighted data cells: `UnitsInStock < 10`. 
-* Strikethrough data rows: `UnitsInStock = 0`. 
-* `UnitPrice` values are set to bold and colored based on the following conditions:  
-  * If a price is lower than the average price, the font is set to red. 
-  * If a price is higher than the average price, the font is set to green. 
-	
-    The average price is calculated within the Blazor Grid's summary item ([DxGridSummaryItem](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGridSummaryItem)).
+![Grid - Customize Element Appearance](images/grid.png)
 
-![Grid - Custom Element Appearance](images/grid.png)
+The example code uses [summary items](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGridSummaryItem) to calculate minimum, maximum, and average column values. Based on cell and summary item values, the [CustomizeElement](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.CustomizeElement) event handler changes cell appearance in the following columns:

```

Comment:
> The example **code**

I understand that you wanted to put extra emphasis on "code" rather than example and basically said "The code of the example" but I believe that "The code example" does exactly the same thing, sounds simpler, and creates a connection to "the example" we mentioned in the previous paragraph.

> Based on cell and summary item values, **the** [CustomizeElement](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.CustomizeElement) event handler changes cell appearance in the following columns:

Here, I'm going to question your article usage. What did you need to say here exactly?

> Based on cell and summary item values, **any** handler of the `CustomizeElement` event changes cell appearance in the following columns:

We can simplify the phrase "a handler of the `CustomizeElement` event" down to just "a `CustomizeElement` event handler" and be done with it.

Therefore, my take on this paragraph is the following one:

```suggestion
The code example uses [summary items](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGridSummaryItem) to calculate minimum, maximum, and average column values. Based on cell and summary item values, a [CustomizeElement](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.CustomizeElement) event handler changes cell appearance in the following columns:
```